tests/integration_tests.rs

```diff
@@ -87,3 +89,74 @@ async fn test_lndk_forwards_onion_message() {
         }
     }
 }
+
+#[tokio::test(flavor = "multi_thread")]
+// Here we test the beginning of the BOLT 12 offers flow. We show that lndk successfully builds an
+// invoice_request and sends it.
+async fn test_lndk_send_invoice_request() {
+    let test_name = "lndk_send_invoice_request";
+    let (_bitcoind, mut lnd, ldk1, ldk2, lndk_dir) =
+        common::setup_test_infrastructure(test_name).await;
+
+    // Here we'll produce a little network path:
+    //
+    // ldk1 <-> ldk2 <-> lnd
```

Comment:
Would be good to have a test where LND isn't connected to the introduction point already.